.drone.star

```diff
@@ -1501,10 +1501,6 @@ def uploadTracingResult(ctx):
         },
         "when": {
             "status": status,
-            "event": [
-                "pull_request",
-                "cron",
-            ],
```

Comment:
this allows tracing to be reported whenever the tests fail. this helps in the investigation of flaky tests.